atm/Sidorchuk/task15/ATM/Card.java

```diff
@@ -0,0 +1,117 @@
+package task15.ATM;
+
+import task15.ATM.money.Currency;
+
+import java.util.Date;
+import java.util.Random;
+
+public class Card {
+
+    private String cardNumber;
+    private String cardCVV;
+    private String cardDateOfExpire;
+    private Person owner;
+    private String pinCode = "1111";
+    private String securityWord;
+    private String status = "valid";
+    private String alarmPinCode = "0000";
+    private BankAccount bankAccount;
+    private Object object;
+
+    public Card(Person owner, Currency currency1, int balance1) {
+        this.owner = owner;
```

Comment:
Again the code duplication. We should avoid that.
You can reuse another constructor at least.